dev_guide/builds.adoc

```diff
@@ -5,19 +5,40 @@
 :icons:
 :experimental:
 :toc: macro
+:toclevels: 3
 :toc-title:
 :prewrap!:
 
 toc::[]
 
 == Overview
-A link:../architecture/core_concepts/builds_and_image_streams.html#builds[build] is a process of creating
-runnable images to be used on OpenShift. There are three build strategies:
+
+A link:../architecture/core_concepts/builds_and_image_streams.html#builds[build]
+is a process of creating runnable images to be used on OpenShift.
```

Comment:
Today a build can output at most one image.
Custom builds can do anything, image-related or not.

I propose repeating what we have in the [core concepts](https://docs.openshift.org/latest/architecture/core_concepts/builds_and_image_streams.html) section:

> A build is the process of transforming input parameters into a resulting object. Most often, the process is used to transform source code into a runnable image.

WDYT?